src/runtime/contrib/tensorrt/tensorrt_builder.cc

```diff
@@ -156,8 +161,18 @@ TensorRTEngineAndContext TensorRTBuilder::BuildEngine() {
   config_ = builder_->createBuilderConfig();
   config_->setMaxWorkspaceSize(max_workspace_size_);
   if (use_fp16_) {
+    config_->setFlag(nvinfer1::BuilderFlag::kGPU_FALLBACK);
```

Comment:
I think this is not needed

Reply:
yes, this is not needed, I will remove it

Reply:
> I think this is not needed

fixed it already